src/content/docs/apis/rest-api-v2/get-started/introduction-new-relic-rest-api-v2.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Introduction to New Relic REST API (v2)
+title: How to Use REST API (v2)
```

Comment:
We like to use sentence case for titles, so: "How to use REST API (v2)".

I prefer slightly "How to use our REST API (v2)"